docker-compose.yml

```diff
@@ -92,10 +92,10 @@ services:
 
   corgi-celery-fast:
     deploy:
-      replicas: 2
+      replicas: 1
```

Comment:
@jasinner Assuming this change matches what's in the ops repo, I'd prefer three workers here so that we get one per AWS Availability Zone (data center). From the MP+ High-Availability guidelines:

"Run at least 2 pod replicas (3 replicas for C1 applications) using a replication controller (Deployment/DeploymentConfig/StatefulSet)."

I don't want to paste an internal link here, but you can search for "MP+ high availability" on The Source.